aggsender/db/aggsender_db_storage.go

```diff
@@ -268,8 +281,40 @@ func (a *AggSenderSQLStorage) SaveOrUpdateCertificate(ctx context.Context, certi
 	return nil
 }
 
+// saveSignedCertificateToFile saves the signed certificate content to a file in the same directory as the database
+// and returns the file path
+func (a *AggSenderSQLStorage) saveSignedCertificateToFile(
+	certificateID common.Hash,
+	signedCertContent string) (string, error) {
+	// Get the directory where the database is stored
+	dbDir := filepath.Dir(a.cfg.DBPath)
+
+	// Create a filename using the certificate ID
+	fileName := fmt.Sprintf("signed_cert_%s.pem", hex.EncodeToString(certificateID[:]))
```

Comment:
Slightly shorter, WDYT?

```suggestion
	fileName := fmt.Sprintf("signed_cert_%s.pem", certificateID.Hex())
```

Reply:
Why is the extension of this file `.pem`? It is not an SSL certificate...
IMO it is more consistent to be either: `.bin`, `.dat` etc. WDYT?

Reply:
Yeah, forgot to change to json, done